testsuite/tests/kuadrant/gateway/test_authpolicy_to_gateway.py

```diff
@@ -0,0 +1,60 @@
+"""Test for AuthPolicy attached directly to gateway"""
+from time import sleep
+import pytest
+
+from testsuite.openshift.objects.auth_config.auth_policy import AuthPolicy
+from testsuite.openshift.objects.gateway_api.route import HTTPRoute
+
+
+@pytest.fixture(scope="module")
+def gateway_httproute(request, gateway, wildcard_domain, module_label, blame):
+    """HTTPRoute with wildcard domain"""
+    route = HTTPRoute.create_instance(gateway.openshift, blame("gw-route"), gateway, {"app": module_label})
+    route.add_hostname(wildcard_domain)
+    route.set_path_match("/")
+
+    request.addfinalizer(route.delete)
+    route.commit()
+    return route
+
+
+@pytest.fixture(scope="module")
+def gateway_authorization(request, gateway, authorization_name, openshift, module_label):
```

Comment:
What if you overide authorization for all gateway tests to be at the Gateway level, same with hostname and route?